build.gradle

```diff
@@ -376,6 +379,22 @@ task setupCIWorkspace {
     }
 }
 
+processResources {
+    onlyIf {
+        // this won't exist on the first time running data gen after wiping the generated folder
+        file('src/main/generated/resources/assets/structurize/lang/default.json').exists()
```

Comment:
hoping the onlyIf is only for the doLast part and not entire processResources

Reply:
Good question - I truly have no idea, but I ran the client and it worked, so I'm assuming the condition only applies to this closure